.github/workflows/chromatic-pr.yaml

```
@@ -58,6 +58,7 @@ jobs:
           externals: |
             **/!(*.module).scss
             packages/components/**/*.css
+            packages/components/**/*.module.css
```

Close! But we actually want the opposite 😂 We want Turbosnap to not be disabled when `.module.css` files are updated, because they are directly imported.

```suggestion
            **/!(*.module).css
```